integration_tests/integration/incidentDetailsEdit.spec.ts

```diff
@@ -318,13 +318,12 @@ context('Incident details (edit) - statement incomplete', () => {
     incidentDetailsPage.timeInputMinutes().type('00')
     incidentDetailsPage.locationSelector().select('Workshop 2')
     incidentDetailsPage.inciteAssociatedPrisonerDeleteButton().click()
-    // TODO: Once the /delete-person page is created, it will redirect automatically to this page so the next line can be removed
-    cy.visit(`/incident-details/G6415GD/34/edit`)
+    cy.get('[data-qa="radio-buttons"]').find('input[value="yes"]').check()
+    cy.get('[data-qa="delete-person-submit"]').click()
     incidentDetailsPage.timeInputHours().should('have.value', '13')
```

Comment:
This seems to be missing the check of the loc.pathname and loc.search, as is done in `incidentDetailSubmiitedEdit.spec.ts`